tests/integration/pilot/common/routing.go

```diff
@@ -1790,7 +1790,7 @@ spec:
 		},
 		setupOpts: fqdnHostHeader,
 		opts: echo.CallOptions{
-			Count: 1,
+			Count: 5 * t.Clusters().Len(),
```

Comment:
isn't there some CallsForMultiNetwork constant somewhere?

Reply:
I thought so too. I can't find anything but `echo.CallsPerWorkload` which is basically static.

Reply:
It's all embedded in the test framework. Just needed to remove `Count`.